### ISSUE TITLE
Python problem

### ISSUE THREAD

**Issue by woodoo46** [EichlerLab/smrtsv2#47]
Hi, 

When I tried to install smrtsv2, I had the following problem

**UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - canu=1.8 -> python[version='3.5.*|>=3.5,<3.6.0a0']

Your python: python=3.6.2**

What should I do to solve it? Thanks!

**Comment by paudano** [comment]
Are you installing this on Linux?

I tried building the dependencies on the latest commit (2.0.1) in case something change, but I am unable to replicate this error. Given the versions, I thought Conda would provide a consistent environment for each install, but it's not as simple as I hoped.

You can edit the canu version in "dep/conda/install_tools.sh". I would try Canu 1.7.1("canu=1.7.1") and see if that solves the problem. Since the tool versions will change from the SMRT-SV version, be sure to keep track of these changes in case you need them to report methods.

Let me know if this help.

**Comment by zihhuafang** [comment]
Hi,

I am also trying to install smrtsv2 on cluster (Linux).
I also ran into the same issue and changing the version of Canu did not solve the problem ( I have tried version 1.7 and 1.9).

I did have canu 1.8 installed on the cluster. I am wondering if I can also export it to the $PATH to solve this issue.

**Comment by paudano** [comment]
In `dep/conda/Makefile`, line 28 is `bin/conda create --yes --name tools python=3.6.2`. Try changing that version to `3.5.3` from `3.6.2`.

To use your Canu install, comment out canu in the install process (see above) and link the canu executable into `dep/bin`.

I'll see if I can do anything to make the Conda builds more robust in future releases. I'm still not sure why I was able to deploy the dependencies with the same build script if the Python version was not supported by Canu.

Let me know if any of this works for you.

**Comment by zihhuafang** [comment]
I managed to make it work by changing the path of canu in dep/Makefile.
It did not work for me when I changed the version of python in the dep/conda/Makefile.

There are still some issues with other conda packages for tools, but I just manually installed it when activate dep/conda/build/envs/tools.

**Comment by paudano** [comment]
Are you able to share the changes you made to get it working? It might be helpful to document it for others who might have similar issues.

**Comment by zihhuafang** [comment]
Sorry for the late reply. I have been busy with other analyses. 

In `dep/Makefile`, I provided my own canu path (I have canu 1.8 installed locally) to generate symlink as followed:         
ln -sf /your/path/to/canu bin/canu

For packages that are installed via conda, I have problem to install gxx_linux-64=7.3.0.
I deleted gxx_linux-64=7.3.0 in the `/dep/conda/install_tools.sh `and installed it manually when activate dep/conda/build/envs/tools environment (using conda activate dep/conda/build/envs/tools). I also checked all the symlinks in `/dep/bin` are correct. If not, I just manually installed the packages that was missing. 

Hope this help for ppl having issues installing the tools.
